tests/dummy/app/pods/select/template.hbs

```diff
@@ -0,0 +1,241 @@
+<div class="section">
+  <div class="title">frost-select</div>
+</div>
+<div class="section">
+  <div class="example">
+    <div class="title">Local filtering</div>
+    <div class="snippet">
+      {{format-markdown "```handlebars
+{{frost-select
+  data=data
+  onChange=(action 'onChange')
```

Comment:
We are also changing all actions to have consistent name.
So `onChange=(action 'onChange')` would become `onChange=(action 'onChangeHandler')`.